app/src/main/java/com/google/samples/apps/topeka/widget/quiz/ToggleTranslateQuizView.java

```diff
@@ -96,10 +96,20 @@ public void setUserInput(Bundle savedInput) {
             initAnswerSpace();
             return;
         }
-        ListAdapter adapter = mListView.getAdapter();
-        for (int i = 0; i < mAnswers.length; i++) {
-            mListView.performItemClick(mListView.getChildAt(i), i, adapter.getItemId(i));
-        }
+
+        mListView.post(new Runnable() {
+            @Override
+            public void run() {
+                ListAdapter adapter = mListView.getAdapter();
```

Comment:
I think that all code in `run()` should be extracted to another method as it is already a quite complex. Give a name like `setupListAdapter() `.